app.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-unused-vars */
+let i = 0;
+let list = [];
+let fromLocalStorage = false;
+function g(j) {
+  return document.getElementById(j);
+}
+function stack(templateString) {
+  const template = document.createElement('template');
+  template.innerHTML = templateString.trim();
+  document.getElementById('books').appendChild(template.content.firstChild);
+}
+function updateLocalStorage() {
+  localStorage.setItem('stuff', JSON.stringify({ arr: list }));
+}
+function addBook() {
+  const title = g('t').value;
+  const author = g('a').value;
+  stack(`<div id="${i + title}">
+      <p>${title}</p>
+      <p>${author}</p>
+      <button id="${i + title}" value="${i}" onclick="removeBook(this.id, this.value)">Remove</button>
+    </div>
+    <hr>`);
+  const locaList = JSON.parse(localStorage.getItem('stuff')).arr;
```

Comment:
- The same problem with arr, you are using it as a function, but it's not, if intend to push it into the list, you need to use push instead